docs/wiki/protocol/design-rationale.md

```diff
@@ -2,8 +2,61 @@
 
 > :warning: This article is a [stub](https://en.wikipedia.org/wiki/Wikipedia:Stub), help the wiki by [contributing](/contributing.md) and expanding it.
 
+
+# Protocol Design Philosophy
+
+These are the core tenets that sparked the work on Ethereum's architeture and imlementation.
+
+### Simplicity
+Since its inception, the Ethereum protocol was designed in a way that reduced complexity and made it simpler, even at the cost of data storage inefficiency and time inefficiency. This stemmed from the idea that any average programmer should ideally be able to understand and implement the entire specification -- this was to minimize the influence on the protocol by an individual or an elite group. While this narrative has transmuted largely due to major changes that have been made to the protocol. 
+
+### Universality 
+
+### Modularity
+
+### Non-discriminant 
+
+### Agility
+
+## Principles
+
 The Ethereum protocol evolves and changes over time but it always follow certain principles. These principles reflect values of the whole community and are reflected in some of the main design decisions of Ethereum.
 
+### Sandwich complexity model 
+
+### Freedom
+
+### Generalization
+
+### We have no features
+
+### Non-risk-aversion
+
+
+## Blockchain level protocol
+
+### Accounts over UTXOs
+
+### Merkle patricia trie
```

Comment:
The MPT and RLP part should be simple, just high level explaining the decision. Both are being replaced, by verkles and ssz, so that could be mentioned as well

Reply:
will keep the wiki as close to the current state of the protocol design 🚀